### ISSUE TITLE
[Bug] 2023.0 Version crashes on Jetson Nano - L4T - Ubuntu 18.04

### ISSUE THREAD

**Issue by mahimairaja** [openvinotoolkit/openvino#18091]
##### System information (version)

- OpenVINO Source=> Runtime - Intel Archive
- OpenVINO Version=> Version 2023.0 
- Operating System / Platform => L4T ( Linux for Tegra - based on Ubuntu 18.04 ) - Nvidia Jetson nano
- Compiler => Cmake
- Problem classification => Model Compilation 
- Device use: => CPU 
- Framework => IR
- Model name => MobileNetV3 - [bin](https://storage.openvinotoolkit.org/repositories/openvino_notebooks/models/mobelinet-v3-tf/FP32/v3-small_224_1.0_float.bin), [xml](https://storage.openvinotoolkit.org/repositories/openvino_notebooks/models/mobelinet-v3-tf/FP32/v3-small_224_1.0_float.xml)

##### Detailed description
The `kernel gets crashed` while compiling the model. So I ran it as script .py file with some logging and interpreter throws the mesage as - `Illegal instruction (core dumped)`

##### Steps to reproduce

1. Create a python virtual environment of version 3.7 and activate it.
2. Download the [zip file](https://storage.openvinotoolkit.org/repositories/openvino/packages/2023.0/linux/l_openvino_toolkit_debian9_2023.0.0.10926.b4452d56304_arm64.tgz)
```bash
sudo wget https://storage.openvinotoolkit.org/repositories/openvino/packages/2023.0/linux/l_openvino_toolkit_debian9_2023.0.0.10926.b4452d56304_arm64.tgz
```
4. Unzip the file and change directory into the folder.
```bash
sudo tar -xf l_openvino_toolkit_debian9_2023.0.0.10926.b4452d56304_arm64.tgz
cd l_openvino_toolkit_debian9_2023.0.0.10926.b4452d56304_arm64
```
5. Install the system requirements 
```bash
sudo -E ./install_dependencies/install_openvino_dependencies.sh
```
6. Activate the openvino environment
```bash
source setupvars.sh
```
You will be promted with `[setupvars.sh] OpenVINO™ environment initialized`.

7. Then clone the openvino/openvino_notebooks repo and run the [001-hello-world notebook](https://github.com/openvinotoolkit/openvino_notebooks/blob/main/notebooks/001-hello-world/001-hello-world.ipynb)

### Related images : 
![2023](https://github.com/openvinotoolkit/openvino/assets/81288263/102f83a7-41e7-463f-b718-f046481295c3)
Figure 1 - Kernel getting crash

# 

![commandline-2023](https://github.com/openvinotoolkit/openvino/assets/81288263/1d0ea038-ee2c-43c3-abc1-9bf12636f2a7)
Figure 2 - Error thrown on model compiling time 

##### Issue submission checklist

 - [x] I report the issue, it's not a question
   
 - [x] I checked the problem with documentation, FAQ, open issues, Stack Overflow, etc and have not found solution
   

 - [x] There is reproducer code and related data files: images, videos, models, etc.

**Comment by avitial** [comment]
@mahimairaja I am not quite sure the Debian archive you are using is compatible with the Ubuntu 18.04 OS you are running. You may have to build from source to get it working, see instructions [here](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build_linux.md). 

For me the _001-hello-world_ notebook seems to be running fine at model compilation stage with the _l_openvino_toolkit_debian9_2023.0.0.10926.b4452d56304_arm64_ archive. My RPI4 board is running Debian 11 (bullseye), and model compilation executes normally without errors on ARM64 (`$ uname -m` returns `aarch64`). 

![2023-07-06-123811_1920x1080_scrot (2)](https://github.com/openvinotoolkit/openvino/assets/20826577/bceec0c2-a2c3-49b7-b57f-e55f81bd5db8)

**Comment by avitial** [comment]
Closing this, I hope previous responses were sufficient to help you proceed. Feel free to reopen and provide additional information or ask any questions related to this topic.

**Comment by ilya-lavrenov** [comment]
Provided actual fix to current master (future 2023.2 reelase)
